Signal/src/view controllers/AdvancedSettingsTableViewController.m

```diff
@@ -18,12 +18,12 @@
 
 @interface AdvancedSettingsTableViewController ()
 
-@property (nonatomic) UITableViewCell *enableCallKitCell;
+@property (nonatomic) UITableViewCell *enableCallKitPrivacyCell;
 @property (nonatomic) UITableViewCell *enableLogCell;
 @property (nonatomic) UITableViewCell *submitLogCell;
 @property (nonatomic) UITableViewCell *registerPushCell;
 
-@property (nonatomic) UISwitch *enableCallKitSwitch;
```

Comment:
We still want to retain the ability to enable CallKit - CallKitPrivacy is sort a "sub feature" relevant only while CallKit is enabled.

Also, at this point, it makes sense for both of these features to be in the "Privacy" settings, not "advanced".